### PR TITLE
#3 knn wine prediction

### PR DESCRIPTION
**Pull request by Szbzltn99**

### CODE REVIEW

**Comment by kadamQ** [comment]
- The code and the plot are good, fix the problems with the tests check!

File: src/knn_model.py [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+        self.x_train = sc_x.fit_transform(self.x_train)
+        self.x_test = sc_x.transform(self.x_test)
+        return self.x_train, self.x_test
+
+    def train(self):
+        """
+        training the model
+        """
+        self.x_train, self.x_test = self.scale()
+        params = {'n_neighbors': [5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16],
```
Comment (Benczus) [inline src/knn_model.py]: please pass the parameter grid as a parameter of the train function

**Review by Benczus — CHANGES_REQUESTED** [review]
You created the branch off of the decision tree branch. You have changed the first branch but not this, so all the problems that were present in the first issue that you later fixed are here. 
You need to pull the changes from the master onto this branch

**Comment by kadamQ** [comment]
### **Resolve the conflicts that this branch has in order to merge it!**

**Comment by kadamQ** [comment]
- The issue was: #3.
- I found the code, the plot, and the results okay.
- Can you check and approve it? @Benczus

**Pull request merged**